src/web_app/js/peerconnectionclient.js

```diff
@@ -29,6 +29,22 @@ var PeerConnectionClient = function(params, startTime) {
     '  constraints: \'' + JSON.stringify(params.peerConnectionConstraints) +
     '\'.');
 
+  if (typeof RTCPeerConnection.generateCertificate === 'function') {
+    trace('Trying to generate an ECDSA certificate.');
+    RTCPeerConnection.generateCertificate({name: 'ECDSA', namedCurve: 'P-256'})
+    .then(function(cert) {
+      if (cert.expires) {
+        trace('Certificate generated successfully.');
+        params.peerConnectionConfig.certificates = [cert];
+      } else {
```

Comment:
Yes it might be, during testing generateCertificate was successful but the RTCCertificate object was empty on M49 and did not contain the `expires` property thus I assumed it failed or it might have succeeded and the property is missing. On M50 the `expires` property is always there.

@henbos, do you know anything about the discrepancy as described above?